Header header          #standard header information

geometry_msgs/Point mcl_mean
std_msgs/Float64MultiArray mcl_cov
std_msgs/Int64 index